canonical-web-and-design/snapcraft.io
# Update design for metadata links on snap page

## Done
Proposed an alternative design for the metadata links on the snap details pages

## QA
- Go to https://snapcraft-io-4022.demos.haus/notion-snap?show_metadata_links=true
- Check the metadata links section looks good
